api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -125,7 +125,7 @@ def test_cannot_delete_if_registration(self):
             pointer_id,
         )
         res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
+        assert_equal(res.status_code, 404)
```

Comment:
Since it is a registration, try to get the node_link of a registration using node route should raise a 404